homework-submission/week1/app.js

```diff
@@ -0,0 +1,109 @@
+//Create a todo list called Static List with three list items:
+
+const staticList = [
+  { description: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+  { description: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+  { description: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+];
+
+//1- Divide the list into reusable components (create at least 2):
+
+// A- reusable list title component:
+//the goal of this component is to create a reusable list title by using props
+class ListTitle extends React.Component {
+  render() {
+    const { title } = this.props;
+    return <h1>{title}</h1>;
+  }
+}
+
+// B- reusable li component
+//the goal of this component is to create a list item
+class Item extends React.Component {
+  render() {
+    // 4- Each list item should have a prop of description and deadline
```

Comment:
Great descriptions already. You're missing one prop here though, besides that it's good to describe the purpose of each prop. The `done` prop has an effect on what the component looks like, good to describe this here so that other developers know what happens when they use this component and its props.